internal/ui/logo.go

```diff
@@ -87,7 +87,7 @@ func (l *Logo) update(msg string, c config.Color) {
 
 func (l *Logo) refreshStatus(msg string, c config.Color) {
 	l.status.SetBackgroundColor(c.Color())
-	l.status.SetText(fmt.Sprintf(l.styles.Body().FgColor.String(), msg))
+	l.status.SetText(fmt.Sprintf("%s%s", l.styles.Body().LogoColorMsg.String(), msg))
```

Comment:
This does not look correct. The text should be decorated here via [fg:bg:xxx] marker.

Reply:
so I am struggling with this.  I am not sure how to convert the hex color to named color.  Any suggestions?

```
func (l *Logo) refreshStatus(msg string, c config.Color) {
	l.status.SetBackgroundColor(c.Color())
        l.status.SetText(fmt.Sprintf("[%s::b]%s", l.styles.Body().LogoColorMsg.String(), msg))
}
```

```
expected: "[white::b]blee\n"
actual  : "[#ffffff::b]blee\n"
```

Reply:
@casonadams Thank you Cason! nw the annotation can either grok color name or hex. BTW Color is a parallel type of the string type.

Reply:
I am not sure if you are happy with what I did. 
Changing test data.

`[white::b]blee\n` -> `[#ffffff:b]blee\n`